app/controllers/Application.scala

```diff
@@ -1,63 +1,68 @@
 package controllers
 
-import com.codahale.metrics.json.MetricsModule
-import com.codahale.metrics.{MetricRegistry, SharedMetricRegistries}
-import com.fasterxml.jackson.databind.{ObjectWriter, ObjectMapper}
-import com.yammer.metrics.reporting.DatadogReporter
 import java.io.StringWriter
 import java.util.concurrent.TimeUnit
-import models.Metrics
-import models.ZkKafka
-import models.ZkKafka._
+
+import com.codahale.metrics.json.MetricsModule
+import com.fasterxml.jackson.databind.{ObjectMapper, ObjectWriter}
+import com.yammer.metrics.reporting.DatadogReporter
+import models.{Metrics, ZkKafka}
+import org.joda.time.DateTime
+import org.joda.time.format.DateTimeFormat
 import play.api.Play.current
 import play.api._
 import play.api.mvc._
-import scala.language.implicitConversions
 
-import org.joda.time.DateTime
-import org.joda.time.format.DateTimeFormat
+import scala.language.implicitConversions
 
 object Application extends Controller {
 
   val validUnits = Some(Set("NANOSECONDS", "MICROSECONDS", "MILLISECONDS", "SECONDS", "MINUTES", "HOURS", "DAYS"))
   val mapper = new ObjectMapper()
 
-  def registryName = Play.configuration.getString("capillary.metrics.name").getOrElse("default")
-  def rateUnit     = Play.configuration.getString("capillary.metrics.rateUnit", validUnits).getOrElse("SECONDS")
-  def durationUnit = Play.configuration.getString("capillary.metrics.durationUnit", validUnits).getOrElse("SECONDS")
-  def showSamples  = Play.configuration.getBoolean("capillary.metrics.showSamples").getOrElse(false)
-  def ddAPIKey     = Play.configuration.getString("capillary.metrics.datadog.apiKey")
+  def registryName: String = Play.configuration.getString("capillary.metrics.name").getOrElse("default")
+
+  def rateUnit: String = Play.configuration.getString("capillary.metrics.rateUnit", validUnits).getOrElse("SECONDS")
```

Comment:
Nit: It would be nice to have these strings as constants rather than inline in multiple places.